### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by jaesharma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,5 +56,18 @@ Simple queueing service built for Internet Computer
 - Send & read messages simultaneously
 - Once processed only
 
+---------
 
+#### <b>DFX Commands for local testing</b>
 
+```
+$ dfx deploy --mode=reinstall --argument '(opt record {principal "<principal-id>"; null})' icsqs
```
Comment (ritvick) [inline README.md]: can you replace this principal id with an actual value taken from dfx identity get-principal ?

**Review by ritvick — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -56,5 +56,18 @@ Simple queueing service built for Internet Computer
...
+#### <b>DFX Commands for local testing</b>
 
+```
+$ dfx deploy --mode=reinstall --argument '(opt record {principal "<principal-id>"; null})' icsqs  
+```
+
+#### <b>To call methods:</b> dfx canister call <canister_id> <method_name> '(<arguments>)'
+examples:
+```
+$ dfx canister call <canister_id> sendMessage '("<message>")'
```
Comment (ritvick) [inline README.md:70]: same is it possible to automate these ?

**Review by ritvick — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jaesharma) [reply README.md:70]: we can use it like this:

for i in `seq 1 100`; do echo “dfx canister call rrkah-fqaaa-aaaaa-aaaaq-cai sendMessage ‘(\“TEST MESSAGE $i\“)‘“; done | bash -

should i add this to readme?

Comment (ritvick) [reply README.md:70]: no need for loop, just one statement is fine

File: README.md [hunk 3]
```diff
@@ -56,5 +56,19 @@ Simple queueing service built for Internet Computer
 - Send & read messages simultaneously
 - Once processed only
 
+---------
 
+#### <b>DFX Commands for local testing</b>
 
+```
+$ dfx deploy --mode=reinstall --argument '(opt record {principal "woicf-w4c3l-ggagd-ktql2-zzwf6-xwrx7-ehupq-an62o-uvf5d-owgvw-nae"; null})' icsqs
```
Comment (ritvick) [inline README.md:64]: I was expecting the principlay to be read from 
$(dfx indetnity get-principal) and use it inside the sting

**Pull request merged**